includes/class-wc-gateway-swedbank-pay-cc.php

```diff
@@ -86,6 +86,12 @@ class WC_Gateway_Swedbank_Pay_Cc extends WC_Payment_Gateway {
 	 */
 	public $logo_url = '';
 
+	/**
+	 * Send cardholder info
+	 * @var string
+	 */
+	public $use_cardholder_info = 'yes';
```

Comment:
The `cardHolder` information is always optional and should only be sent if it is available. It doesn't make sense requiring this information, I think.

Reply:
@asbjornu `cardHolder` is optional, you're right. We can close this PR.

Reply:
@aait, is `cardholder` optional in the plugin as well? It's important that the plugin doesn't make fields required that aren't required in our API. `cardholder` and `riskIndicator` are such fields (with sub-fields) that are optional and should only be sent if there is data available to send.